osu.Game.Rulesets.Taiko/Mods/TaikoModSingleTap.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Localisation;
+
+namespace osu.Game.Rulesets.Taiko.Mods
+{
+    public class TaikoModSingleTap : TaikoInputBlockingMod
+    {
+        public override string Name => @"Single Tap";
+        public override string Acronym => @"SG";
+        public override LocalisableString Description => @"You must only use one key!";
+
```

Comment:
this is speculative and id like a second opinion since the added code isn't much regardless but maybe there should be a base mod class for single tap between rulesets to avoid this duplicated code

Reply:
I thought of this too, but I'm unsure of how effective this would be given that almost all areas of the code had *some* kind of change to it that was Taiko-specific

Reply:
I think the potential harm outweights the benefit. Extracting a base class for three strings is not much gain, and having them separate prevents the possibility of overcomplicated class hierarchies popping up.

If there is a class to commonise, then it can be done after the pattern of exactly what is needed is clear, not before.